cmd/minikube/cmd/start.go

```diff
@@ -1205,55 +1204,7 @@ func getKubernetesVersion(old *config.MachineConfig) (string, bool) {
 		exit.WithCodeT(exit.Data, `Unable to parse "{{.kubernetes_version}}": {{.error}}`, out.V{"kubernetes_version": paramVersion, "error": err})
 	}
 	nv := version.VersionPrefix + nvs.String()
-
-	if old == nil || old.KubernetesConfig.KubernetesVersion == "" {
-		return nv, isUpgrade
-	}
-
-	oldestVersion, err := semver.Make(strings.TrimPrefix(constants.OldestKubernetesVersion, version.VersionPrefix))
-	if err != nil {
-		exit.WithCodeT(exit.Data, "Unable to parse oldest Kubernetes version from constants: {{.error}}", out.V{"error": err})
-	}
-	defaultVersion, err := semver.Make(strings.TrimPrefix(constants.DefaultKubernetesVersion, version.VersionPrefix))
-	if err != nil {
-		exit.WithCodeT(exit.Data, "Unable to parse default Kubernetes version from constants: {{.error}}", out.V{"error": err})
-	}
-
-	if nvs.LT(oldestVersion) {
```

Comment:
Where'd these warnings go?

Reply:
I'm trying to determine if we still need to pull images if we have a version upgrade on a preexisting cluster -- just waiting to see how `TestVersionUpgrade` responds to this.